### ISSUE TITLE
Gitea Plugin

### ISSUE THREAD

**Issue by Dual-0** [geluk/matrix-webhook-gateway#21]
Hello,

I tried the gitea.ts plugin on my hosted gitea but webhook-gateway shows the error:
`2021-10-16 10:00:46.810.000 INFO [plg-gitea] Unknown Gitea action: 'undefined'`

gitea config:
![image](https://user-images.githubusercontent.com/37346547/137583486-585cd68f-6505-414e-a758-3f80159c6db6.png)

last webhook:
![image](https://user-images.githubusercontent.com/37346547/137583549-d21837af-d260-4054-8c91-eef6307e7962.png)

do you need more informations to debug?

regards

**Comment by geluk** [comment]
Perhaps the request body is formatted in an unexpected manner? Can you try replacing the following line:
```
this.logger.info(`Unknown Gitea action: '${body.action}'`);
```
With this?
```
this.logger.info(`Gitea webhook body: '${JSON.stringify(body)}'`);
```

**Comment by geluk** [comment]
That definitely does help, it looks like the 'secret' field may be missing sometimes. I've made it an optional field.

You can pull the latest Docker image for the `:dev` tag to try it out. Could you let me know if this fixes your issue?

**Comment by geluk** [comment]
Fixed in version 1.1.
